source/install/prod-ubuntu.rst

```diff
@@ -95,48 +100,88 @@ Set up Database Server
 ------------------------
 
-1. For the purposes of this guide we will assume this server has an IP
+For the purposes of this guide we will assume this server has an IP
    address of ``10.10.10.2``
-2. For the sake of making this guide simple we located the files at
-   ``/home/ubuntu/mattermost``. In the future we will give guidance for
-   storing under ``/opt``.
-3. We have also elected to run the Mattermost Server as the ``ubuntu``
-   account for simplicity. We recommend setting up and running the
-   service under a ``mattermost`` user account with limited permissions.
+
+2. Create a mattermost user and group
+
+    - ``sudo adduser --system --group mattermost``
+
+3. Change to the mattermost home directory.
+
+    ``cd /home/mattermost``
+
 4. Download `any version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing:
 
    -  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
-   -  Where ``vX.X.X`` is typically the latest Mattermost release version, which is currently ``v3.3.0``. 
-   
-5. Unzip the Mattermost Server by typing:
+   -  Where ``vX.X.X`` is the latest version.
+
+5. Extract the Mattermost Server files by typing:
+
+   -  ``sudo tar -xvzf *.gz``
+
+6. Change the user and group of the extracted files to mattermost
 
-   -  ``tar -xvzf mattermost-X.X.X-linux-amd64.tar.gz``
+   - ``sudo chown -R mattermost:mattermost mattermost/``
 
-6. Create the storage directory for files. We assume you will have
+7. Create the storage directory for files. We assume you will have
    attached a large drive for storage of images and files. For this
    setup we will assume the directory is located at
    ``/mattermost/data``.
 
    -  Create the directory by typing:
    -  ``sudo mkdir -p /mattermost/data``
-   -  Set the ubuntu account as the directory owner by typing:
-   -  ``sudo chown -R ubuntu /mattermost``
+   -  Set the mattermost account as the directory owner by typing:
+   -  ``sudo chown -R mattermost:mattermost /mattermost``
 
 7. Configure Mattermost Server by editing the config.json file at
-   ``/home/ubuntu/mattermost/config``
+   ``/home/mattermost/config``
 
-   -  ``cd ~/mattermost/config``
+   -  ``cd /home/mattermost/mattermost/config``
    -  Edit the file by typing:
    -  ``vi config.json``
-   -  replace ``DriverName": "mysql"`` with ``DriverName": "postgres"``
-   -  replace
-      ``"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"``
-      with
-      ``"DataSource": "postgres://mmuser:mmuser_password@10.10.10.1:5432/mattermost?sslmode=disable&connect_timeout=10"``
-   -  Optionally you may continue to edit configuration settings in
+   - If you are using PostgreSQL:
+      1. Set ``DriverName":`` to ``"postgres"``
+      2. Set ``"DataSource:"`` to the following value: ``"postgres://mmuser:mmuser_password@10.10.10.1:5432/mattermost?sslmode=disable&connect_timeout=10"``
```

Comment:
at the top of this section we say: **For the purposes of this guide we will assume this server has an IP
     address of ``10.10.10.2``**

but here we are using `10.10.10.1` which is never mentioned